sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/PropagateEmptyRelationSuite.scala

```diff
@@ -67,6 +68,28 @@ class PropagateEmptyRelationSuite extends PlanTest {
     comparePlans(optimized, correctAnswer)
   }
 
+  test("remove empty relation children from Union") {
```

Comment:
Although this is an improvement, shall we add `SPARK-32241: `?

Reply:
Fixed, thanks.